src/components/InputSet/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Input } from '../../variables';
+import { User } from '../../Shared';
+import styles from './styles.module.css';
+
+interface InputsStatesSignature extends User {
+  [key: string]: string;
```

Comment:
Why do you need an indexer here?

Reply:
to address object's fields with dynamic key

Reply:
I think that static fields set is better than dynamic. Will you add here many different fields that you can't describe with interface?